provider/oauth2/tests.py

```diff
@@ -597,6 +597,25 @@ def test_access_token_response_valid_token_type(self):
         token = self._login_authorize_get_token()
         self.assertEqual(token['token_type'], constants.TOKEN_TYPE, token)
 
+    def test_client_credentials_grant(self):
+        response = self.client.post(self.access_token_url(), {
+            'grant_type': 'client_credentials',
+            'client_id': self.get_client().client_id,
+            'client_secret': self.get_client().client_secret,
+        })
+
+        self.assertEqual(200, response.status_code, response.content)
```

Comment:
In the success case, verify that:
1. An `access_token` is returned (with an `expires_in` field).
2. A `refresh_token` is NOT returned.